test/requests/inlay_hints_expectations_test.rb

```diff
@@ -0,0 +1,14 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+require "expectations/expectations_test_runner"
+
+class InlayHintsExpectationsTest < ExpectationsTestRunner
+  expectations_tests RubyLsp::Requests::InlayHints, "inlay_hints"
+
+  def run_expectations(source)
+    document = RubyLsp::Document.new(source)
+    RubyLsp::Requests::InlayHints.new(document, 0..5).run
```

Comment:
If we use `default_args`, you shouldn't need to override `run_expectations`. Similar to [here](https://github.com/Shopify/ruby-lsp/blob/29c1d14020ca74f1f92f46708245b386630b91ec/test/requests/selection_ranges_expectations_test.rb#L21), we can do
```ruby
def default_args
  [0..5]
end
```